packages/cra-template-typescript/template/README.md

```diff
@@ -1,3 +1,5 @@
+# Getting Started with Create React App
+
```

Comment:
[MD041 - First line in file should be a top level heading](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md041---first-line-in-file-should-be-a-top-level-heading)